samples/README.md

```diff
@@ -297,6 +301,15 @@ View the [source code](https://github.com/googleapis/python-storage/blob/main/sa
 
 `python storage_create_bucket_class_location.py`
 
+-----
+### Create Bucket Notifications
+[![Open in Cloud Shell][shell_img]](https://console.cloud.google.com/cloudshell/open?git_repo=https://github.com/googleapis/python-storage&page=editor&open_in_editor=samples/snippets/storage_create_bucket_notifications.py,samples/README.md)
+
+View the [source code](https://github.com/googleapis/python-storage/blob/main/samples/snippets/storage_create_bucket_notifications.py). To run this sample:
+
+
+`python storage_create_bucket_notifications.py`
```

Comment:
The code uses sysargs to determine the bucket and topic, so I think all of these readme examples should mention that. For instance `...bucket_notifications.py your-bucket your-topic-name`

Reply:
Done! Will file a separate PR for all other readme examples :)